src/components/createBook/BookForm.tsx

```diff
@@ -141,28 +225,34 @@ function CreateBook() {
                 getOptionValue={(option: AuthorResponse) => option.Id.toString()}
                 onChange={handleSelectedAuthorsChange}
               />
-              <button className='add-btn' onClick={() => setShowModal((s) => !s)}><AddIcon className = "icon" size={40} color="#fce4db" /></button>
+              <button className='add-btn' onClick={() => setShowModal((show) => !show)}><AddIcon className = "icon" size={40} color="#fce4db" /></button>
               { showModal && <CreateAuthor setIsAuthorsChanged={setIsAuthorsChanged} closeModal={() => setShowModal(false)}/>}
             </div>
           </div>
           <div className='section'>
             <label className="date-label" >Publish date</label>
-            <input className = "form-input" type="date"
-              onChange={(e) => setBook(prevState => ({ ...prevState, PublishDate: e.target.value }))}
+            <input className = "form-input" type="date" value={book.PublishDate ? new Intl.DateTimeFormat('en-CA').format(new Date(book.PublishDate)) : ''}
```

Comment:
when you have more than 3 attributes break into multiple lines it will be more readable.